File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L6/06/fungsi.py
import sys

def terbalik(angka):
    arr = []
    while angka > 0:
        sisa = angka % 10
        arr.append(sisa)
        angka //= 10
    for digit in arr:
        print(digit, end='')
    print()

if __name__ == "__main__":
    print("Enter an integer: ", end='')
    input_num = int(input())
    terbalik(input_num)

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L6/06/fungsi.kt
package IR

import java.util.ArrayList
import java.util.Scanner

fun main() {
    println("Enter an integer: ")
    val sc = Scanner(System.`in`)
    val input = sc.nextInt() // input integer
    terbalik(input) // memanggil fungsi reverse
}

fun terbalik(angka: Int) {
    val arr = ArrayList<Int>()
    var tempAngka = angka
    while (tempAngka > 0) { // looping sampai angka habis
        val sisa = tempAngka % 10 // membagi bilangan tersebut dengan 0 sampai tidak ada hasil bagi
        arr.add(sisa)
        tempAngka /= 10
    }
    for (i in arr.indices) {
        print(arr[i])
    }
    println()
}

